src/components/scenes/Fio/FioRequestConfirmationScene.tsx

```diff
@@ -211,10 +211,7 @@ export class FioRequestConfirmationConnected extends React.Component<Props, Stat
       } catch (error: any) {
         this.setState({ loading: false })
         this.resetSlider()
-        showError(lstrings.fio_request_error_header)
-        showDevError(
-          `${lstrings.fio_request_error_header}. ${error.json && error.json.fields && error.json.fields[0] ? JSON.stringify(error.json.fields[0].error) : ''}`
-        )
+        showError(error.json?.fields?.[0]?.error ?? '')
```

Comment:
Let's do it this way:

```ts
const errorText = error.json?.fields?.[0]?.error ?? String(error)
showError(lstrings.fio_request_error_header + '. ' + errorText)
```

That way we still show something if JSON isn't present. Or, we could do this if we really don't want `lstrings.fio_request_error_header`:

```ts
showError(error.json?.fields?.[0]?.error ?? error)
```